easybuild/easyconfigs/l/LCov/LCov-1.13-GCCcore-7.2.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'LCov'
+version = '1.13'
+
+homepage = 'http://ltp.sourceforge.net/coverage/lcov.php'
+description = "LCOV - the LTP GCOV extension"
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
```

Comment:
@rubendibattista You'll need to switch this back to `http://downloads.sourceforge.net/ltp/`, the `/ltp` is probably preventing a correct download via the `SOURCEFORGE_SOURCE` template...

Reply:
@boegel - @rubendibattista fixed it by using Github instead, which works fine in my testing.